docs/source/tasks/plr.rst

```diff
@@ -0,0 +1,11 @@
+Pupillary light reflex (PLR)
```

Comment:
```suggestion
.. _PR:

Pupillary light reflex (PLR)
```

Reply:
the ``.. _PR:`` at the top of this file will create an internal hyperlink, so that you can do :ref:`PR` in another RsT file, and sphinx will automatically create a hyperlink to this PLR task file, in the built documentation.

Reply:
Changes implemented and pushed